File: ueueo-framework/ueueo-blob-storing/src/main/java/Abp/BlobStoring/BlobContainerExtensions.cs
using System.IO;
using System.Linq;
using System.Threading;
using System.Threading.Tasks;

namespace Volo.Abp.BlobStoring;

public static class BlobContainerExtensions
{
    public static void SaveAsync(
        this IBlobContainer container,
        String name,
        byte[] bytes,
        boolean overrideExisting = false,
        CancellationToken cancellationToken = default
    )
    {
        using (var memoryStream = new MemoryStream(bytes))
        {
            container.SaveAsync(
                name,
                memoryStream,
                overrideExisting,
                cancellationToken
            );
        }
    }

    public static  Task<byte[]> GetAllBytesAsync(
        this IBlobContainer container,
        String name,
        CancellationToken cancellationToken = default)
    {
        using (var stream = container.GetAsync(name, cancellationToken))
        {
            return stream.GetAllBytesAsync(cancellationToken);
        }
    }

    public static  Task<byte[]> GetAllBytesOrNullAsync(
        this IBlobContainer container,
        String name,
        CancellationToken cancellationToken = default)
    {
        var stream = container.GetOrNullAsync(name, cancellationToken);
        if (stream == null)
        {
            return null;
        }

        using (stream)
        {
            return stream.GetAllBytesAsync(cancellationToken);
        }
    }
}

File: ueueo-frameworks/ueueo-blob-storing/src/main/java/com/ueueo/blobstoring/BlobContainerExtensions.java
package com.ueueo.blobstoring;

import com.ueueo.threading.CancellationToken;
import org.apache.commons.io.IOUtils;

import java.io.ByteArrayInputStream;
import java.io.IOException;
import java.io.InputStream;

public class BlobContainerExtensions {

    public static void save(
            IBlobContainer container,
            String name,
            byte[] bytes,
            boolean overrideExisting,
            CancellationToken cancellationToken
    ) {
        InputStream memoryStream = new ByteArrayInputStream(bytes);
        container.save(name,
                memoryStream,
                overrideExisting,
                cancellationToken);
    }

    public static byte[] getAllBytes(
            IBlobContainer container,
            String name,
            CancellationToken cancellationToken) {
        InputStream stream = container.get(name, cancellationToken);
        try {
            return IOUtils.toByteArray(stream);
        } catch (IOException e) {
            return null;
        } finally {
            try {
                stream.close();
            } catch (IOException e) {
                e.printStackTrace();
            }
        }
    }

    public static byte[] getAllBytesOrNull(
            IBlobContainer container,
            String name,
            CancellationToken cancellationToken) {
        InputStream stream = container.get(name, cancellationToken);
        if (stream == null) {
            return null;
        }

        try {
            return IOUtils.toByteArray(stream);
        } catch (IOException e) {
            return null;
        } finally {
            try {
                stream.close();
            } catch (IOException e) {
                e.printStackTrace();
            }
        }
    }
}
